src/components/NotFound/NotFound.js

```diff
@@ -99,7 +93,7 @@ class NotFound extends React.Component<Props> {
     return (
       <ModalDialog
         className={classList.join(' ')}
-        isVisible={this.props.isVisible}
+        isVisible={true}
```

Comment:
0: hmm :)

Reply:
😄 Should we use default props in `ModalDialog`?

Reply:
All is fine, nothing to see here 💃